### PR TITLE
Documentation on Marketing Software Integration

### PR DESCRIPTION
**Pull request by GANGANI**
## Purpose
Add the documentation on Marketing Software Integration

### CODE REVIEW

File: en/docs/learn/enterprise-marketing-software.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Enterprise Marketing Software Integration
+
+If your business uses or is willing to use enterprise marketing softwares such as Hubspot or Salesforce, WSO2 IAM solution has the ability to interact with those systems while giving you the simplicity to manage your organizations&#39; identities from a single place.
+
+This topic provides instructions on how to integrate HubSpot with WSO2 Identity Server to provision users from WSO2 Identity Server. After completing this tutorial you can see the users you add using WSO2 Identity Server being created in HubSpot and lead information updates in WSO2 Identity Server being updated in HubSpot as well.
+
+## Configuring HubSpot
+
+1. **Test accounts are created and managed within developer accounts.** If you don&#39;t have a developer account in HubSpot, you can sign up from [here](https://app.hubspot.com/signup/developers?_ga=2.39153443.1802613489.1576611619-500942594.1573763828).
```
Comment (emswbandara) [inline en/docs/learn/enterprise-marketing-software.md:9]: add a new line before starting the statemet 'If you don&#39;t have a developer account..'

File: en/mkdocs.yml [hunk 2]
```diff
@@ -98,6 +98,7 @@ nav:
           - 'Provisioning Overview': learn/provisioning-tutorials.md
           - 'Inbound Provisioning': learn/configuring-inbound-provisioning.md
           - 'Outbound Provisioning': learn/configuring-outbound-provisioning.md
+            - 'Enterprise Marketing Software': learn/enterprise-marketing-software.md
```
Comment (emswbandara) [inline en/mkdocs.yml:101]: let's change this to "Provisioning users to Hubspot"

In the introduction of the doc, you can say hubspot is an EMS..

File: en/docs/learn/enterprise-marketing-software.md (hunk 1 above)
Comment (GANGANI) [reply en/docs/learn/enterprise-marketing-software.md]: Fixed https://github.com/wso2/docs-is/pull/1758/commits/154049c67d80d52fb96b2c1bc8ea421436b12aae

File: en/mkdocs.yml (hunk 2 above)
Comment (GANGANI) [reply en/mkdocs.yml]: Fixed https://github.com/wso2/docs-is/pull/1758/commits/154049c67d80d52fb96b2c1bc8ea421436b12aae

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    ```
+
+    ![scim-provisioning-conf](../assets/img/tutorials/scim-provisioning-conf.png)
+
+1. Click Update and save the configuration
+
+#### Step 03: Configuring the outbound provisioning
+
+Now we need to enable sending out provisioning requests to the previously configured provisioning system for any user 
+management operation done via the management console, SOAP API or the SCIM API. To do that, you must configure outbound
```
Comment (gomathyk) [inline en/docs/learn/provisioning-users-to-hubspot.md:103]: SOAP API, or the SCIM API.

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+    ![scim-provisioning-conf](../assets/img/tutorials/scim-provisioning-conf.png)
+
+1. Click Update and save the configuration
+
+#### Step 03: Configuring the outbound provisioning
+
+Now we need to enable sending out provisioning requests to the previously configured provisioning system for any user 
+management operation done via the management console, SOAP API or the SCIM API. To do that, you must configure outbound 
+provisioning identity providers against the resident service provider. So, based on the outbound configuration,
```
Comment (gomathyk) [inline en/docs/learn/provisioning-users-to-hubspot.md:104]: the outbound provisioning

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+1. Click Update and save the configuration
+
+#### Step 03: Configuring the outbound provisioning
+
+Now we need to enable sending out provisioning requests to the previously configured provisioning system for any user 
+management operation done via the management console, SOAP API or the SCIM API. To do that, you must configure outbound 
+provisioning identity providers against the resident service provider. So, based on the outbound configuration, 
+users created in WSO2 Identity Server can also be provisioned to external systems like HubSpot or Google Apps.
+
+1. Log on to the [Management Console](https://localhost:9443/carbon/) using the username and password.
```
Comment (gomathyk) [inline en/docs/learn/provisioning-users-to-hubspot.md:107]: Log in

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+2. In the **Main** menu under the **Identity** section, click **Resident** under **Service Providers.**
+3. In the resulting screen, expand the **Outbound Provisioning Configuration** section.
+4. In the **Outbound Provisioning Configuration** section, do the following.
+
+    1. Select the identity provider you added from the drop-down menu available and click the + sign to add it. If you 
+    have not added an identity provider yet, this step is not possible.
+    2. Once added, the identity provider is displayed as an entry in the list. Select scim from the drop-down to ensure 
+    that the SCIM operation is used for provisioning.
+    3. If the option, Blocking is enabled, the outbound provisioning request will be blocked until the response is
```
Comment (gomathyk) [inline en/docs/learn/provisioning-users-to-hubspot.md:116]: Boldify the following: 
**Blocking**
**Enable Rules**
**Update**

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+    ![create-new-account](../assets/img/tutorials/create-new-account.png)
+
+1. Enter the **First Name** and **LastName**.
+2. Enter a preferred password in the **Password** and **Confirm password** text boxes.
+3. Enter your email address in the **Email** text box and the phone number in the Phone **Numbers** text box.
+4. You may enter other details in the given text boxes.
+5. Read the Privacy Policy and select the **Privacy Policy** checkbox.
+6. Click **Register**.
+7. Login to HubSpot.
+8. On the navigation panel, click contacts. You will see that the user you created in WSO2 IS has also been created in
```
Comment (gomathyk) [inline en/docs/learn/provisioning-users-to-hubspot.md:146]: **Contacts**

Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed https://github.com/wso2/docs-is/pull/1758/commits/d61920ada49084e1d9a38ac45aceaf204f1c7b36

File: en/docs/learn/provisioning-users-to-hubspot.md (hunk 6 above)
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md (hunk 5 above)
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md (hunk 4 above)
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md (hunk 3 above)
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+
+    ```
+        Username: admin
+        Password: admin
+        User Endpoint: https://localhost:8253/hubspot/contacts
+    ```
+
+    ![scim-provisioning-conf](../assets/img/tutorials/scim-provisioning-conf.png)
+
+1. Click Update and save the configuration
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+    ```
+
+## Configuring Identity Server
+
+#### Step 01: Set up Identity server and configure a required claim
+
+1. Download [WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+2. Navigate to `<IS_HOME>\bin` and start the server by executing one of the following commands.
+
+    ```
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md:63]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+       <EndPointsList xmlns:ns1="http://endpoints">
+       <APIKEY>7xxxxxxxxxxxxxxxxxxxxxxxxxxxx8</APIKEY>
+       </EndPointsList> 
+    ```
+
+## Configuring Identity Server
+
+#### Step 01: Set up Identity server and configure a required claim
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+This topic provides instructions on how to integrate HubSpot with WSO2 Identity Server to provision users from WSO2 
+Identity Server. After completing this tutorial you can see the users you add using WSO2 Identity Server being created 
+in HubSpot and lead information updates in WSO2 Identity Server being updated in HubSpot as well.
+
+## Configuring HubSpot
+
+1. **Test accounts are created and managed within developer accounts.** 
+        
+      If you don&#39;t have a developer account in HubSpot, you can sign up from
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
+# Provisioning users to Hubspot
+
+If your business uses or is willing to use enterprise marketing softwares such as Hubspot or Salesforce, WSO2 IAM 
+solution has the ability to interact with those systems while giving you the simplicity to manage your 
+organizations&#39; identities from a single place.
+
+This topic provides instructions on how to integrate HubSpot with WSO2 Identity Server to provision users from WSO2 
+Identity Server. After completing this tutorial you can see the users you add using WSO2 Identity Server being created
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

File: en/docs/learn/provisioning-users-to-hubspot.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
+# Provisioning users to Hubspot
+
+If your business uses or is willing to use enterprise marketing softwares such as Hubspot or Salesforce, WSO2 IAM 
+solution has the ability to interact with those systems while giving you the simplicity to manage your 
+organizations&#39; identities from a single place.
```
[the remark this answers is not in the document]
Comment (GANGANI) [reply en/docs/learn/provisioning-users-to-hubspot.md]: Fixed d61920a

**Pull request merged**